website/docs/user_guides/overview_of_chaos_dashboard.md

```diff
@@ -0,0 +1,150 @@
+---
+id: overview_of_chaos_dashboard
+title: Overview of Chaos Dashboard
+---
+
+This document will give you an overview of Chaos Dashboard. We will simplify some of the steps we saw in [Run Chaos Experiment](run_chaos_experiment) and guide you to use the visual interface based on our existing features.
```

Comment:
The overview runs the same experiment as in the doc of `Run Chaos Experiment`. Maybe we should keep these two paragraphs at the same time?

Because I wanna show through comparison that some parts of the dashboard are easy to use.